Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,55 +1,64 @@
+  clickHandler = event => {
+    let toUpdate = {...this.state}
+    toUpdate.screen = event.target.innerText;
+    this.setState = ({
+      screen: toUpdate
+    });
+    console.log(event.target.innerText, toUpdate);
   };
+  render() {
   return (
     <div className="container">
       <div className="layerOne">
-        <CalculatorDisplay output={myState.screen.output} />
+        <CalculatorDisplay output={this.state.screen} />
       </div>
       <div className="layerTwo">
         <ActionButton
-          click={clickHandler}
-          style={style.flex3}
+          click={this.clickHandler}
+          style={this.style.flex3}
           symbol={"Clear"}
         />
-        <ActionButton click={clickHandler} style={style.flex2Red} symbol={"÷"} />
+        <ActionButton click={this.clickHandler} style={this.style.flex2Red} symbol={"÷"} />
       </div>
       <div className="layerThree">
-        <NumberButton click={clickHandler} style={style.flex2} number={"7"} />
-        <NumberButton click={clickHandler} style={style.flex2} number={"8"} />
-        <NumberButton click={clickHandler} style={style.flex2} number={"9"} />
-        <ActionButton click={clickHandler} style={style.flex2Red} symbol={"x"} />
-        <NumberButton click={clickHandler} style={style.flex2} number={"4"} />
-        <NumberButton click={clickHandler} style={style.flex2} number={"5"} />
-        <NumberButton click={clickHandler} style={style.flex2} number={"6"} />
-        <ActionButton click={clickHandler} style={style.flex2Red} symbol={"-"} />
-        <NumberButton click={clickHandler} style={style.flex2} number={"1"} />
-        <NumberButton click={clickHandler} style={style.flex2} number={"2"} />
-        <NumberButton click={clickHandler} style={style.flex2} number={"3"} />
-        <ActionButton click={clickHandler} style={style.flex2Red} symbol={"+"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"7"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"8"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"9"} />
+        <ActionButton click={this.clickHandler} style={this.style.flex2Red} symbol={"x"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"4"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"5"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"6"} />
+        <ActionButton click={this.clickHandler} style={this.style.flex2Red} symbol={"-"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"1"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"2"} />
+        <NumberButton click={this.clickHandler} style={this.style.flex2} number={"3"} />
+        <ActionButton click={this.clickHandler} style={this.style.flex2Red} symbol={"+"} />
```

Comment:
for future improvements try using .map() generate new components and pass props to them. This is how You create reusable components.